Source/WebCore/css/CSSFontFeatureValuesRule.idl

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+typedef USVString CSSOMString;
+
+[
+    Exposed=Window
+] interface CSSFontFeatureValuesRule : CSSRule {
+    readonly attribute sequence<CSSOMString> fontFamilies;
```

Comment:
Do we differ from Gecko and the spec intentionally here?

```
attribute CSSOMString fontFamily;
```

is what I see here. I assume it is an implementation detail?

Reply:
https://www.w3.org/TR/css-fonts-4/#cssfontfeaturevaluesrule

I've changed it to be closer to the spec (even if I think it should be plural)